templates/agent/src/handler/ens.ts

```diff
@@ -128,13 +128,12 @@ export async function handleEns(
       };
     }
     const data = await getUserInfo(address);
-    let txUrl = `${baseTxUrl}/transaction/?transaction_type=send&buttonName=Tip%20${data?.ensDomain ?? ""}&amount=1&token=USDC&receiver=${
-      isAddress(address) ? address : data?.address
-    }`;
-    console.log(txUrl);
+
+    let sendUrl = `${txpayUrl}/?&amount=1&token=USDC&receiver=${address}`;
```

Comment:
@humanagent you can remove the `isAddress` import above from `viem` also since you removed the usage here (installed from this template for ETHGlobal and noticed the unused import ;) )